programs/svm-spoke/src/instructions/bundle.rs

```diff
@@ -81,6 +81,11 @@ impl RelayerRefundLeaf {
     pub fn to_bytes(&self) -> Vec<u8> {
         let mut bytes = Vec::new();
 
+        // This requires the first 64 bytes to be 0 within the encoded leaf data. This protects any kind of EVM leaf
+        // from ever being used on SVM. Note that the chain_id field in theory should protect this but this 64 blank
+        // slot protects it under all cases.
+        bytes.extend_from_slice(&[0u8; 64]);
+
         bytes.extend_from_slice(&self.amount_to_return.to_le_bytes());
```

Comment:
Not directly related to this PR, but only now I realize we might been serializing this wrong below. Particularly, nowhere do we encode the lengths of the vectors, we just concatenate them. I think instead of implementing our custom serialization we should have used Anchor's `serialize` that uses Borch encoding where before each vector there is also a 4 byte length inserted.